<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeatSelection</class>
 <widget class="QDialog" name="SeatSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Odabir sedista</string>
  </property>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>nazad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Projection">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>16</y>
     <width>591</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>591</width>
     <height>381</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>589</width>
      <height>379</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_Seats">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_Price_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Price">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_Seats_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_Reserve">
        <property name="text">
         <string>Rezervisi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Buy">
        <property name="text">
         <string>Kupi</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Email">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>424</y>
     <width>204</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Name">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>346</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Email">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>219</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ime za rezervaciju</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Email">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Info email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Info">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rezervacije nisu moguce, do filma je ostalo manje od 30 minuta</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
